eng/source-build-patches/0027-Remove-libraries-packages.proj-harvesting.patch

```diff
@@ -0,0 +1,27 @@
+From 7e470148fdae7a3a69a39a1c0de41034bb57926c Mon Sep 17 00:00:00 2001
+From: Davis Goodin <dagood@microsoft.com>
+Date: Mon, 2 Nov 2020 11:54:57 -0600
+Subject: [PATCH] Remove libraries-packages.proj harvesting
+
+Remove some prebuilts by removing package harvesting. Package harvesting
+is not compatible with the concept of a prebuiltless source-build.
+---
+ src/libraries/libraries-packages.proj | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/libraries/libraries-packages.proj b/src/libraries/libraries-packages.proj
+index cfbf75d8a4e..603d523df55 100644
+--- a/src/libraries/libraries-packages.proj
++++ b/src/libraries/libraries-packages.proj
+@@ -25,7 +25,7 @@
+   <Import Project="$(PkgDir)baseline\baseline.props" />
+ 
+   <!-- Calculate PackageDownload items for restore. -->
+-  <Import Project="$(RepositoryEngineeringDir)restore\harvestPackages.targets" Condition="'$(MSBuildRestoreSessionId)' != ''" />
++  <Import Project="$(RepositoryEngineeringDir)restore\harvestPackages.targets" Condition="'$(MSBuildRestoreSessionId)' != '' and '$(DotNetBuildFromSource)' != 'true'" />
```

Comment:
@ViktorHofer can't this be removed in main?

Reply:
Not yet. The "old" packaging system still compares against a baseline version and this code path downloads these baselines up-front. The code block can be removed after all pkgprojs in src/libraries are removed.